ui/src/pages/backoffice/Acquisition/Order/OrderDetails/OrderInformation.js

```diff
@@ -31,7 +31,7 @@ export class OrderInformation extends React.Component {
     order.status === 'CANCELLED' &&
       rightTable.push({ name: 'Cancel reason', value: order.cancel_reason });
     return (
-      <Grid columns={2}>
+      <Grid columns={2} id="order-info">
```

Comment:
For all the following occurrences, is there a particular reason with went with `id` instead of `class`?

Reply:
yes because it is for the scrolling menu and we would like to have there unique id, so if you click on payment info it does not jump to vendor randomly